modules/andes-core/broker/src/main/java/org/wso2/andes/amqp/QpidAndesBridge.java

```diff
@@ -160,7 +160,7 @@ public static void messageReceived(IncomingMessage incomingMessage, AndesChannel
      * @param andesMessage Message coming in
      * @param andesChannel AndesChannel
      */
-    public static void messageReceived(AndesMessage andesMessage, AndesChannel andesChannel) {
+    public static void messageReceived(AndesMessage andesMessage, AndesChannel andesChannel) throws AndesException {
```

Comment:
Update the doc comment as well with throws part

Reply:
:+1: